### DISCUSSION TITLE
Antd Radio.Group defaultValue don't work when wrapper with Controller, what's wrong with my code?

### DISCUSSION THREAD

**Discussion by bonniePxx** [react-hook-form/react-hook-form#3516] [Q&A] [answered]
import { Button, Radio } from "antd";
 <Controller
          as={
            <Radio.Group defaultValue='ww' name='alg'>
              <Radio value='ww' key={1}>
                {3333}
              </Radio>
            </Radio.Group>
          }
	name='alg'
        control={control}
	defaultValue={defaultAlg}
/>

**Comment by ghostd** [comment, marked as answer]
Hi

If you're using the last version of react-hook-form, you can test the `render` of the [`Controller` component](https://react-hook-form.com/api#Controller).

**Comment by bluebill1049** [comment]
Thank you @ghostd you are right.
